src/web/views/Components/Table/Table.vue

```diff
@@ -1,80 +1,420 @@
+        />
       </div>
     </div>
   </main>
 </template>
 
 <script>
+import Code from "../../Elements/Code/Code.vue";
 import ComponentCode from "../../Elements/ComponentCode/ComponentCode.vue";
-import BIMDataTable from "../../../../../src/BIMDataComponents/BIMDataTable/BIMDataTable.vue";
-import BIMDataInput from "../../../../../src/BIMDataComponents/BIMDataInput/BIMDataInput.vue";
 import BIMDataCheckbox from "../../../../../src/BIMDataComponents/BIMDataCheckbox/BIMDataCheckbox.vue";
-
+import BIMDataInput from "../../../../../src/BIMDataComponents/BIMDataInput/BIMDataInput.vue";
+import BIMDataTable from "../../../../BIMDataComponents/BIMDataTable/BIMDataTable.vue";
+import AgeCustomCell from "./AgeCustomCell.vue";
+import CountryCustomCell from "./CountryCustomCell.vue";
 
 export default {
   components: {
+    Code,
     ComponentCode,
-    BIMDataTable,
+    BIMDataCheckbox,
     BIMDataInput,
-    BIMDataCheckbox
+    BIMDataTable,
+    AgeCustomCell,
+    CountryCustomCell,
   },
   data() {
     return {
-      compensated: false,
-      rowData: [
-        ["First name", "Last name", "Country"],
-        ["John", "Doe", "Germany"],
-        ["Jane", "Doe", "Austria"],
-      ],
+      simpleExample: {
+        columns: ["First name", "Last name", "Age", "Country"],
+        rows: [
+          ["John", "Doe", 26, "Germany"],
+          ["Jane", "Doe", 21, "Austria"],
+          ["Martine", "Durand", 35, "France"],
+          ["Giuseppe", "Bompiani", 64, "Italy"],
+          ["Enrico", "Fermi", 41, "Italy"],
+          ["Lev Davidovitch", "Landau", 23, "Russia"],
+        ],
+        rowHeight: 50,
+        selectable: false,
+        selection: [],
+        paginated: false,
+        perPage: 4,
+      },
+
+      advancedExample: {
+        columns: [
+          { id: "fullName", label: "Name" },
+          { id: "age", label: "Age", width: "40px" },
+          { id: "country", label: "Country", width: "200px", align: "center" },
+        ],
+        rows: [
+          { firstName: "John", lastName: "Doe", age: 26, country: "Germany" },
+          { firstName: "Jane", lastName: "Doe", age: 21, country: "Austria" },
+          {
+            firstName: "Martine",
+            lastName: "Durand",
+            age: 35,
+            country: "France",
+          },
+          {
+            firstName: "Giuseppe",
+            lastName: "Bompiani",
+            age: 64,
+            country: "Italy",
+          },
+          { firstName: "Enrico", lastName: "Fermi", age: 41, country: "Italy" },
+          {
+            firstName: "Lev Davidovitch",
+            lastName: "Landau",
+            age: 23,
+            country: "Russia",
+          },
+        ],
+      },
+
+      // Props docmentation
       propsData: [
-        ["Props", "Type", "Description", "Example"],
+        [
+          "Props",
+          "Type",
+          "Required",
+          "Default value",
+          "Description",
+          "Example",
```

Comment:
Just to say something: I feel like there is no example in the props documentation ?